docs/getting_started/go/run_workers_with_cloud_go/index.md

```diff
@@ -215,8 +224,8 @@ Before running your application, set the following environment variables with th
   <TabItem value="macos" label="MacOS">
 
 ```bash
-export TEMPORAL_MTLS_TLS_CERT=/path/to/tls/cert.pem
-export TEMPORAL_MTLS_TLS_KEY=/path/to/tls/key.pem
+export TEMPORAL_MTLS_TLS_CERT=C:\path\to\tls\cert-file.(pem|crt)
```

Comment:
This set of code is for MacOS, original path should be maintained

Reply:
Fixed.

Reply:
This still seems to be windows paths, did you perhaps forget to push your changes?

Reply:
Sorry about that. Was trying to decide with Brian whether to finish this as a single ticket or do all SDKs as separate tickets. Changes are still local on my system.